x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,110 @@ export const PolicyList = React.memo(() => {
+            <>
+              <EuiFlexItem grow={false}>
+                <PolicyLink name={item.name} route={routePath} href={routeUrl} />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiText
+                  color="subdued"
+                  size="xs"
+                  style={{ marginLeft: '6px', whiteSpace: 'nowrap' }}
+                >
+                  <FormattedMessage
+                    id="xpack.siem.endpoint.policyList.revision"
+                    defaultMessage="rev. {revNumber}"
+                    values={{ revNumber: item.revision }}
+                  />
+                </EuiText>
+              </EuiFlexItem>
+            </>
+          );
         },
         truncateText: true,
       },
       {
-        field: 'revision',
-        name: i18n.translate('xpack.siem.endpoint.policyList.revisionField', {
-          defaultMessage: 'Revision',
+        field: 'created_by',
+        name: i18n.translate('xpack.siem.endpoint.policyList.createdBy', {
+          defaultMessage: 'Created By',
         }),
-        dataType: 'number',
+        truncateText: true,
       },
       {
-        field: 'package',
-        name: i18n.translate('xpack.siem.endpoint.policyList.versionField', {
-          defaultMessage: 'Version',
+        field: 'created_at',
+        name: i18n.translate('xpack.siem.endpoint.policyList.createdAt', {
+          defaultMessage: 'Created Date',
         }),
-        render(pkg) {
-          return `${pkg.title}  v${pkg.version}`;
+        render(createdAt: string) {
+          return <FormattedDateAndTime date={new Date(createdAt)} />;
         },
       },
       {
-        field: 'description',
-        name: i18n.translate('xpack.siem.endpoint.policyList.descriptionField', {
-          defaultMessage: 'Description',
+        field: 'updated_by',
+        name: i18n.translate('xpack.siem.endpoint.policyList.updatedBy', {
+          defaultMessage: 'Last Updated By',
         }),
         truncateText: true,
       },
       {
-        field: 'config_id',
-        name: i18n.translate('xpack.siem.endpoint.policyList.agentConfigField', {
-          defaultMessage: 'Agent Configuration',
+        field: 'updated_at',
+        name: i18n.translate('xpack.siem.endpoint.policyList.updatedAt', {
+          defaultMessage: 'Last Updated',
         }),
-        render(version: string) {
-          return (
-            <LinkToApp
-              data-test-subj="agentConfigLink"
-              appId="ingestManager"
-              appPath={`#/configs/${version}`}
-              href={`${services.application.getUrlForApp('ingestManager')}#/configs/${version}`}
-            >
-              {version}
-            </LinkToApp>
-          );
+        render(updatedAt: string) {
+          return <FormattedDateAndTime date={new Date(updatedAt)} />;
         },
       },
+      {
+        field: 'package',
+        name: i18n.translate('xpack.siem.endpoint.policyList.versionField', {
```

Comment:
```suggestion
        name: i18n.translate('xpack.siem.endpoint.policyList.versionFieldLabel', {
```
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types